freeipa.spec.in

```diff
@@ -1126,12 +1126,9 @@ if [  $? -eq 0 ]; then
     # work with the network being offline
     /bin/systemctl start network-online.target
```

Comment:
Should we be still restarting FreeIPA and thus trying to force the upgrade?

Reply:
Yes we still want to force the upgrade.

This change is to fix the narrow case of a distro upgrade in Fedora where the networking may not be available. In likely all other cases the network will be up and the upgrade will proceed more or less as usual but instead of calling the upgrade separately it will be called as part of the restart.

You make a good point about the logging, I'll need to give that some thought. It may be better to actually fork out to ipa-server-upgrade so the logging is done properly.